app/stylesheets/pages/_vericite.scss

```diff
@@ -0,0 +1,98 @@
+@import "base/environment";
```

Comment:
This file is the same as the turnitin other than variable renames.  This also in its current form potentially breaks some a11y color choices.
The turnitin file should just be used.